src/main/java/org/fundacionjala/trello/driver/SharedDriver.java

```diff
@@ -0,0 +1,19 @@
+package org.fundacionjala.trello.driver;
+
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.WebDriver;
+
+import org.fundacionjala.trello.config.Environment;
+
+public class SharedDriver {
+
+    public SharedDriver() {
+        if (DriverFactory.getDriver() == null) {
+            Environment env = Environment.getInstance();
+            WebDriver driver = BrowserFactory.getBrowser(env.getBrowserName());
+            driver.manage().window().setSize(new Dimension(Environment.getInstance().getBrowserWidth(),
```

Comment:
I suggest you call all Environment methods in the same way.
Just use `env` as an instance, or call them directly from the class

Reply:
Done